pkg/agent/openflow/pipeline.go

```diff
@@ -293,7 +293,7 @@ func GetFlowTableID(tableName string) uint8 {
 func GetTableList() []binding.Table {
 	tables := make([]binding.Table, 0)
 	for _, obj := range tableCache.List() {
-		t := obj.(binding.Table)
+		t := obj.(*Table).ofTable
```

Comment:
this looks like a bug? Was this function (`GetTableList`) not used by anything else?

Reply:
Yeah, I think it was a bug, this function is only used in `antctl get ovsflows`, it probably never worked before.